pkg/scaling/resolver/hashicorpvault_handler.go

```diff
@@ -170,9 +175,163 @@ func (vh *HashicorpVaultHandler) Read(path string) (*vaultapi.Secret, error) {
 	return vh.client.Logical().Read(path)
 }
 
+func (vh *HashicorpVaultHandler) Write(path string, data map[string]interface{}) (*vaultapi.Secret, error) {
```

Comment:
Would be nice, if we can add documentation for funcitons defined in this file. With a short description on what it is doing. Thanks!